Database/Database.sql

```diff
@@ -0,0 +1,344 @@
+/* CREATE DATABASE */
+DROP DATABASE IF EXISTS suitapp_db;
+CREATE DATABASE suitapp_db;
+\c suitapp_db;
+
+/* CREATE TABLES */
+CREATE TABLE users
+(
+    id SERIAL PRIMARY KEY,
+    firstname VARCHAR (20) NOT NULL,
+    lastname VARCHAR (20) NOT NULL,
+    username VARCHAR (20) UNIQUE NOT NULL,
+    user_password VARCHAR (30) NOT NULL,
+    email VARCHAR (30) UNIQUE NOT NULL,
+    avatar_url TEXT,
+    bio VARCHAR (500),
+    ui_theme BOOLEAN NOT NULL DEFAULT TRUE,
+    time_created DATE NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE posts
+(
+    id SERIAL PRIMARY KEY,
+    image_url TEXT NOT NULL,
+    caption VARCHAR,
+    owner_id INT REFERENCES users(id) ON DELETE CASCADE,
+    hashtag_str TEXT,
+    time_created DATE NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE comments
+(
+    id SERIAL PRIMARY KEY,
+    post_id INT REFERENCES posts(id),
+    commenter_id INT REFERENCES users(id),
+    comment_body VARCHAR,
+    time_created DATE NOT NULL DEFAULT CURRENT_TIMESTAMP 
+);
+
+CREATE TABLE likes
+(
+    id SERIAL PRIMARY KEY,
+    post_id INT REFERENCES posts(id),
+    comment_id INT REFERENCES comments(id),
+    liker_id INT REFERENCES users(id),
+    time_created DATE NOT NULL DEFAULT CURRENT_TIMESTAMP 
+);
+
+CREATE TABLE follows
+(
+    id SERIAL PRIMARY KEY,
+    follower_id INT REFERENCES users(id),
+    followed_user_id INT REFERENCES users(id),
+    time_created DATE NOT NULL DEFAULT CURRENT_TIMESTAMP 
+);
+
+CREATE TABLE emojis
```

Comment:
We need to discuss again this table, now we talked about what (before leaving the conference room) I see it in different way

Reply:
Just let me know what changes should be made to the table, and I'll start working on seeding that data as well!